std/hints.go

```diff
@@ -7,14 +7,13 @@ import (
 	"github.com/consensys/gnark/std/math/bits"
 )
 
-// GetHints return std hints that are always injected in gnark solvers
-func GetHints() []hint.Function {
-	return []hint.Function{
-		sw_bls24315.DecomposeScalar,
-		sw_bls12377.DecomposeScalar,
-		bits.NTrits,
-		bits.NNAF,
-		bits.IthBit,
-		bits.NBits,
-	}
+// RegisterHints register std hints that are always injected in gnark solvers
+func RegisterHints() {
+	// note that importing these packages may already triggers a call to hint.Register(...)
+	hint.Register(sw_bls24315.DecomposeScalar)
```

Comment:
Even though `hint.Register` does not overwrite already registered hints, then it would be more safe to guard these registrations with a `sync.Once`.